pom.xml

```diff
@@ -56,6 +56,7 @@ under the License.
     <module>flink-ml-iteration</module>
     <module>flink-ml-lib</module>
     <module>flink-ml-tests</module>
+    <module>flink-ml-lib</module>
```

Comment:
It looks like this line is not needed anymore.

Could you manually do `mvn clean spotless:apply package` and make sure this PR could pass all checks?

Reply:
OK, I will do it later.

Reply:
done